src/session/session_api.c

```diff
@@ -2480,6 +2500,12 @@ __open_session(WT_CONNECTION_IMPL *conn, WT_EVENT_HANDLER *event_handler, const
      */
     WT_RELEASE_WRITE_WITH_BARRIER(session_ret->active, 1);
 
+    /* Set the default helper error */
+    char *init_err_msg;
+    WT_ERR(__wt_malloc(session_ret, 1 * sizeof(char), &init_err_msg));
```

Comment:
It would be good to make this code more robust to any change in the initial default string. Eg define the default string as a macro or constant, then use the length + 1 to allocate memory before then copying the string.

Reply:
Done in [this commit](https://github.com/wiredtiger/wiredtiger/pull/11327/commits/e8af544bc85c2c2359707e14f8cbdfca419dd370)